service-base/src/main/java/fi/nls/oskari/domain/map/OskariLayer.java

```diff
@@ -28,7 +28,7 @@
 
     private boolean isBaseMap = false;
     private boolean isInternal = false;
-    private int dataproviderId;
+    private Integer dataproviderId;
```

Comment:
I might end up removing the dataproviderId variable completely on the admin refactoring but it's ok for now.